frontend/src/components/UI/textChipsOutline/TextChipsOutline.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from './TextChipsOutline.module.scss';
+
+interface ITextChipsOutline {
```

Comment:
imo It is better to put interfaces in a separate file, which will be located in the directory of your component.